site/src/pages/OrganizationSettingsPage/OrganizationProvisionersPage/ProvisionerRow.tsx

```diff
@@ -0,0 +1,173 @@
+import type {
+	ProvisionerDaemon,
+	ProvisionerDaemonStatus,
+} from "api/typesGenerated";
+import {
+	StatusIndicator,
+	StatusIndicatorDot,
+	type StatusIndicatorProps,
+} from "components/StatusIndicator/StatusIndicator";
+import { TableCell, TableRow } from "components/Table/Table";
+import { ChevronDownIcon, ChevronRightIcon } from "lucide-react";
+import { JobStatusIndicator } from "modules/provisioners/JobStatusIndicator";
+import {
+	ProvisionerTag,
+	ProvisionerTags,
+	ProvisionerTruncateTags,
+} from "modules/provisioners/ProvisionerTags";
+import { useState, type FC } from "react";
+import { cn } from "utils/cn";
+import { relativeTime } from "utils/time";
+import { ProvisionerVersion } from "./ProvisionerVersion";
+import { ProvisionerKey } from "pages/OrganizationSettingsPage/OrganizationProvisionersPage/ProvisionerKey";
+
+const variantByStatus: Record<
+	ProvisionerDaemonStatus,
+	StatusIndicatorProps["variant"]
+> = {
+	idle: "success",
+	busy: "pending",
+	offline: "inactive",
+};
+
+type ProvisionerRowProps = {
+	provisioner: ProvisionerDaemon;
+	buildVersion: string | undefined;
+};
+
+export const ProvisionerRow: FC<ProvisionerRowProps> = ({
+	provisioner,
+	buildVersion,
+}) => {
+	const [isOpen, setIsOpen] = useState(false);
+
+	return (
+		<>
+			<TableRow>
+				<TableCell>
+					<button
```

Comment:
native html `button`, no component?

Reply:
I used the native because I wanted a button "without any styles" but I think I can do that easily in another way or creating a "unstyled" variant for those cases and keep the styles for focus.